modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -756,6 +808,22 @@ The amount of memory to give an instance of a Data Transform (Wasm) virtual mach
 
 ---
 
+=== data_transforms_read_buffer_memory_percentage
```

Comment:
Should be marked as for internal use only:

https://github.com/redpanda-data/documentation-private/issues/2358

Reply:
done